src/components/NewWorkspaceModal.ts

```diff
@@ -445,12 +523,12 @@ const NewWorkspaceModal = withDisplayName(
                           div({ style: { marginBottom: '0.2rem' } }, ['Inherited groups:']),
                           ...existingGroups.join(', '),
                         ]),
-                      h(Select, {
+                      h(Select<string, true>, {
```

Comment:
This is… odd. It's a map of things that are aways the group name -> true?

Reply:
Select's second type parameter indicates whether or not it allows selecting multiple values.
https://github.com/DataBiosphere/terra-ui/blob/25c33ccd6260d2ee9d1859698474e494540bc8e9/src/components/common/Select.ts#L160-L166

Reply:
But `isMulti` is a parameter that is passed. It seems very odd that it is a top-level argument as well. But oh well, I see that `isMulti` is then removed from the props.

Reply:
react-select's type definitions require that the `isMulti` prop matches the `IsMulti` type parameter.
https://github.com/JedWatson/react-select/blob/2f94e8d228ea32dbd0faa1f7685a67b74c70420f/packages/react-select/src/Select.tsx#L191